ingestion/src/metadata/ingestion/source/database/hive/metastore_dialects/mysql/__init__.py

```diff
@@ -1,23 +0,0 @@
-#  Copyright 2021 Collate
-#  Licensed under the Apache License, Version 2.0 (the "License");
-#  you may not use this file except in compliance with the License.
-#  You may obtain a copy of the License at
-#  http://www.apache.org/licenses/LICENSE-2.0
-#  Unless required by applicable law or agreed to in writing, software
-#  distributed under the License is distributed on an "AS IS" BASIS,
-#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-#  See the License for the specific language governing permissions and
-#  limitations under the License.
-"""
-Hive Metastore Mysql Dialect
-"""
-from sqlalchemy.dialects import registry
-
-from .dialect import HiveMysqlMetaStoreDialect
-
-__version__ = "0.1.0"
-registry.register(
-    "hive.mysql",
-    "metadata.ingestion.source.database.hive.metastore_dialects.mysql.dialect",
-    "HiveMysqlMetaStoreDialect",
-)
```

Comment:
@pmbrull  these files would be required to load the dialect

Reply:
thanks! Any reason for them to be inside `__init__`?

Reply:
since they are used when getting the connection maybe we can move this logic to `connection` instead of doing the import?

Reply:
actually I'll add the `__init__.py` back under these directories